obspy/imaging/tests/test_taup_geo.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
This contents of this file should be moved to the `test_taup_geo.py` file in the `obspy.taup` package.

Reply:
Actually the contents of this file already seem to be in `obspy/taup/tests/test_taup_geo.py` in master.. so I'll also remove it